stdlib/FileWatching/test/runtests.jl

```diff
@@ -32,7 +35,9 @@ for i in 1:n
     if !fd_in_limits && Sys.islinux()
         run(`ls -la /proc/$(getpid())/fd`)
     end
-    @test fd_in_limits
+    if ismacos_arm
```

Comment:
Should this be `!ismacos_arm`? Does this test really only work on macOS ARM?

Reply:
Yeah, that was a typo. :-/